### ISSUE TITLE
'imagenet-rest.py' cannot use pre-trained resNet model

### ISSUE THREAD

**Issue by chenchen718** [ppwwyyxx/tensorpack#433]
I load the provided .npz resNet pre-trained model and try continuing training (or finetuning) the network. But I find the network trains from scratch rather than using the loaded model. Both training and validation errors at the beginning are over 98%. 

What might be wrong with this?

**Comment by ppwwyyxx** [comment]
Assuming you are using the right model with the right command, you may need a much smaller learning rate to start with.

**Comment by chenchen718** [comment]
@ppwwyyxx  Thank you very much! I change the learning rate to a very small one, and now everything becomes normal.
